.github/workflows/ci.yml

```diff
@@ -2,16 +2,15 @@ name: CI
 on:
   push:
     branches:
-      - "actions/**"
-      - "wip/**"
       - "main"
   pull_request:
     branches:
-      - "**"
+      - "main"
```

Comment:
As above, our CI build time is quiet long so this is a conservation measure that we only build things with intent to merge to main. Branch-to-branch merging is not a common workflow for us.